t/t2802-uri-cmd.t

```diff
@@ -31,7 +31,7 @@ test_expect_success 'flux-uri passes through ssh and local URIs unchanged' '
 	test "$result" = "$local_uri" &&
 	result=$(flux uri --remote $local_uri) &&
 	test_debug "echo flux uri --remote $local_uri returns $result" &&
-	test "$result" = "ssh://$(hostname -s)/tmp/flux-xyzzy/local-0"
+	test "$result" = "ssh://$(hostname)/tmp/flux-xyzzy/local-0"
```

Comment:
Nice catch! The test should probably accept either long or short hostname, but since the code that builds the remote uri is probably using the same result as `$(hostname)` I think your fix will work.